src/pages/Analysis/Analysis.types.ts

```diff
@@ -0,0 +1,16 @@
+import { Option } from "../../context/game-context.types";
+
+export type ErrorMessage = {
+  success: Boolean;
```

Comment:
define types in small case only. Somewhere you use `Boolean` started with capital `B` and somewhere you use type `number` started with small `n`